fuzz/fuzz_nfa.c

```diff
@@ -553,22 +553,6 @@ nfa_regress_minimise_false_positive(void)
 }
 
 static bool
-segfault_fsm_trim(void)
```

Comment:
This should have had "regress_" in the name; it's a regression test for the crash.

Reply:
I had considered renaming it. But I don't think I want to keep around specially crafted tests for constraints on API contracts, because those ought to come out through ~fuzzing~ property based testing in general.